peerconnection.go

```diff
@@ -31,7 +31,8 @@ package webrtc
 
 /*
 #cgo CXXFLAGS: -std=c++0x
-#cgo LDFLAGS: -L${SRCDIR}/lib
+#cgo linux LDFLAGS: -L${SRCDIR}/lib -z noexecstack
+#cgo darwin LDFLAGS: -L${SRCDIR}/lib
```

Comment:
I'm not necessarily opposed to these changes but do these flags need to be set at the source level?

Would something like this,

`CGO_LDFLAGS_ALLOW="-z|noexecstack" go build -ldflags '-z noexecstack'`

work here?

https://gitweb.torproject.org/builders/tor-browser-build.git/tree/projects/snowflake/build#n46

Reply:
Actually, even simpler, you can set
`export CGO_LDFLAGS="-z noexecstack"`
I think we should do this for the Tor Browser build.

It worries me that the stack is executable by default, but this is probably something that should be fixed with the go linker rather than here. The advantage to doing everything in environment variables or with `-ldflags` as you suggested is you don't need to remember to set things in more than one place.

I'm definitely good if the decision is to not merge this pull request.